dev-requirements.txt

```diff
@@ -2,114 +2,139 @@
 # This file is autogenerated by pip-compile with python 3.6
 # To update, run:
 #
-#    pip-compile --output-file=dev-requirements.txt dev-requirements.in
+#    pip-compile --extra=dev --output-file=dev-requirements.txt setup.py
 #
 attrs==22.2.0
     # via
-    #   -c requirements.txt
+    #   cattrs
+    #   ddtrace
     #   jsonschema
     #   pytest
+bytecode==0.13.0
+    # via ddtrace
+cattrs==1.0.0
+    # via ddtrace
+cdislogging==1.1.1
+    # via indexd (setup.py)
+certifi==2023.5.7
+    # via requests
+charset-normalizer==2.0.12
+    # via requests
 click==8.0.4
-    # via
-    #   -c requirements.txt
-    #   flask
+    # via flask
 coverage[toml]==6.2
     # via pytest-cov
-dataclasses==0.8 ; python_version < "3.7"
-    # via
-    #   -c requirements.txt
-    #   werkzeug
+dataclasses==0.8
+    # via werkzeug
```

Comment:
dataclasses is being added by werkzug **because** python 3.6 is being used still.
👎🏻